app/models/agents/trigger_agent.rb

```diff
@@ -11,7 +11,7 @@ class TriggerAgent < Agent
 
       The `type` can be one of #{VALID_COMPARISON_TYPES.map { |t| "`#{t}`" }.to_sentence} and compares with the `value`.  Note that regex patterns are matched case insensitively.  If you want case sensitive matching, prefix your pattern with `(?-i)`.
 
-      The `value` can be a single value or an array of values. In the case of an array, if one or more values match then the rule matches.
+      The `value` can be a single value or an array of values (all values must be stored as strings). In the case of an array, if one or more values match then the rule matches. *Warning*: avoid using `field!=value` with arrays, you should use `not in` instead.
```

Comment:
More complete re-write:

The `value` can be a single value or an array of values. In the case of an array, all items must be strings, and if one or more values match, then the rule matches. Note: avoid using `field!=value` with arrays, you should use `not in` instead.